orc8r/gateway/python/magma/common/sentry.py

```diff
@@ -164,38 +119,22 @@ def _filter_excluded_messages(event: Event, hint: Hint, patterns_to_exclude: Lis
     return event
 
 
-def _get_before_send_hook(
-        sentry_status: SentryStatus,
-        patterns_to_exclude: List[str],
-) -> Optional[SentryHook]:
+def _get_before_send_hook(patterns_to_exclude: List[str]) -> Optional[SentryHook]:
 
     def filter_excluded_and_unmarked_messages(
             event: Event, hint: Hint,
     ) -> Optional[Event]:
-        event = _ignore_if_not_marked(event)
+        event = _ignore_if_marked(event)
         if event and patterns_to_exclude:
             return _filter_excluded_messages(event, hint, patterns_to_exclude)
         return None
 
-    def filter_excluded_messages(
-            event: Event, hint: Hint,
-    ) -> Optional[Event]:
-        return _filter_excluded_messages(event, hint, patterns_to_exclude)
-
-    if sentry_status == SentryStatus.SEND_SELECTED_ERRORS:
-        return filter_excluded_and_unmarked_messages
-    if patterns_to_exclude:
-        return filter_excluded_messages
-    return None
+    return filter_excluded_and_unmarked_messages
```

Comment:
Should this now not be `filter_excluded_and_marked_messages`? Also in line 127.

Reply:
Good catch, fixed.